server/src/routes.rs

```diff
@@ -0,0 +1,38 @@
+use super::sql;
+use std::collections::HashMap;
+
+use actix_web::{get, HttpResponse};
+
+pub struct State {
+    pub db_conn: sqlx::SqlitePool,
+}
+
+type Data = actix_web::web::Data<State>;
+
+#[derive(serde::Serialize)]
+struct ErrorJsonResp<'m> {
+    msg: &'m str,
+    details: String,
```

Comment:
I'd prefer `detail` since `s` usually indicates an array

Reply:
Fixed in 3decf66a67deceb6545ee74e7e294062d030c271